File: examples/petstore/oseg_generated/csharp/UpdatePetWithFormDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class UpdatePetWithFormDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            new PetApi(config).UpdatePetWithForm(
                petId: 12345,
                name: "Pet's new name",
                status: "sold"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Pet#UpdatePetWithForm: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/petstore/oseg_generated/php/UpdatePetWithFormDefaultExample.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    (new OpenAPI\Client\Api\PetApi(config: $config))->updatePetWithForm(
        pet_id: 12345,
        name: "Pet's new name",
        status: "sold",
    );
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Pet#updatePetWithForm: '
        . print_r($e->getResponseObject());
}
